src/controllers/userController.js

```diff
@@ -0,0 +1,70 @@
+import userService from '../services/userService';
+import { jwtSignUser } from '../utils/index';
+import { comparePassword } from '../helpers/comparePassword';
+import { logins } from '../models';
+/**
+ * @class UsersController
+ */
+class UserController {
+  /** Login User
+   * @static
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @returns {string} loginUsers
+   */
+  static async loginAUser(req, res) {
+    const { email, password } = req.body;
+    try {
+      const loggedUser = await userService.loginAUser(email);
+      if (!loggedUser) {
+        return res.status(404).json({
+          error: 'Email does not exit',
```

Comment:
This is a sensitive error response. Giving the user clue of your database email. Rather than saying email does not exist while not say `invalid login credentials`. So in this case the user would be left to think is it my password that is invalid or my email ?.

Reply:
Fixed

Reply:
Fixed, thank you